dist/docker-images/ziti-controller/compose.test.bash

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env bash
+
+# exec this script to test the checked-out Docker controller and router deployments
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o xtrace
+
+cleanup(){
+	docker compose --profile test down --volumes --remove-orphans
+}
+
+BASEDIR="$(cd "$(dirname "${0}")" && pwd)"
+REPOROOT="$(cd "${BASEDIR}/../../.." && pwd)"
+cd "${REPOROOT}"
+
+[[ -s ./go.work ]] && {
+	echo "ERROR: remove go.work before testing deployment" >&2
+	exit 1
+}
+
+: "${ZIGGY_UID:=$(id -u)}"
+: "${ZITI_GO_VERSION:=$(grep -Po '^go\s+\K\d+\.\d+(\.\d+)?$' ./go.mod)}"
+
+export COMPOSE_FILE=\
+./dist/docker-images/ziti-controller/compose.yml\
+:./dist/docker-images/ziti-controller/compose.test.yml\
+:./dist/docker-images/ziti-router/compose.yml\
+:./dist/docker-images/ziti-router/compose.test.yml \
+ZIGGY_UID \
+ZITI_GO_VERSION \
+ZITI_PWD="ziggypw" \
+ZITI_CTRL_ADVERTISED_ADDRESS="ctrl1.127.21.71.0.sslip.io" \
+ZITI_CTRL_ADVERTISED_PORT="1280" \
+ZITI_CONTROLLER_IMAGE="ziti-controller:local" \
+ZITI_ROUTER_IMAGE="ziti-router:local" \
+ZITI_ROUTER_NAME="router1"
+
+export ZITI_ROUTER_ADVERTISED_ADDRESS="${ZITI_ROUTER_NAME}.127.21.71.0.sslip.io" \
+ZITI_ENROLL_TOKEN="/home/ziggy/.config/ziti/${ZITI_ROUTER_NAME}.jwt"
+
+mkdir -p ./release/amd64/linux
```

Comment:
why not use a /tmp dir? alternatively, i don't think there's a need for "amd64" in this path, is there?

Reply:
It's an artifact of the symbols used by `gox` for OS and architecture, and the patterns used by our build that wraps `gox`. 

The release workflow builds container images by stuffing the release ziti binary into the image, avoiding the need to re-build it. That Dockerfiles used in that workflow adopt the same `gox` variables and patterns to locate the binary.

This test workflow avoids changing the Dockerfiles by further extending the pattern, e.g., the directory structure.

TL;DR Some things resemble the release workflow because the test is designed to exercise the release, not to express a minimal pattern for its own sake.